sdk/include/psdk/chstring.h

```diff
@@ -6,125 +6,372 @@
 #include <windows.h>
 #include <provexce.h>
 
+/*
+ * Framedyn dates from ancient times when wchar_t was a typedef to unsigned short.
+ * In order be able to use newer compilers, we define inline overloaded wrappers.
+ */
+using CHSTRING_WCHAR=unsigned short ;
+using CHSTRING_LPCWSTR=const CHSTRING_WCHAR*;
+using CHSTRING_LPWSTR=CHSTRING_WCHAR*;
+static_assert(sizeof(CHSTRING_WCHAR) == sizeof(wchar_t), "CHSTRING_WCHAR must be of same size than wchar_t");
```

Comment:
```suggestion
typedef unsigned short CHSTRING_WCHAR;
typedef const CHSTRING_WCHAR *CHSTRING_LPCWSTR;
typedef CHSTRING_WCHAR *CHSTRING_LPWSTR;
C_ASSERT(sizeof(CHSTRING_WCHAR) == sizeof(wchar_t));
```

Reply:
Why ?

Reply:
This is not an attempt to anything. And modern here mostly means "compilant to a 10-year-old standard". I think we settled this a while ago...